64.md

```diff
@@ -0,0 +1,65 @@
+NIP-64
+======
+
+Inbox model
+-----------
+
+`draft` `optional`
+
+Suppose **Sarah** wants to subscribe to the notes **Walter** writes.
+
+She must let Walter know she is following him. The first step is to find Walter's _inbox list_:
+
+```jsonc
+{
+  "kind": 10064,
+  "pubkey": "<walter>",
+  "tags": [
+    ["relay", "wss://walter.inbox"]
+  ]
+}
+```
+
+Now Sarah can publish a `kind:6401` _follow intent_ to `wss://walter.inbox` tagging Walter:
```

Comment:
Why not use kind 10 and 11 events? https://github.com/nostr-protocol/nips/pull/349

Reply:
These kinds are already being used by NIP-29.

Reply:
This could be useful https://undocumented.nostrkinds.info/

Reply:
6401 is within the reserved Nostr Job Result range. :facepalm: